ci/scripts/prepare-cached-bitstream.sh

```diff
@@ -12,11 +12,10 @@ set -ex
 readonly TOPLEVEL=top_earlgrey
 readonly TOPLEVEL_BIN_DIR="${BIN_DIR}/hw/${TOPLEVEL}"
 readonly TARGETS=(
-  //hw/bitstream:test_rom
-  //hw/bitstream:rom
-  //hw/bitstream:rom_otp_dev
-  //hw/bitstream:rom_mmi
-  //hw/bitstream:otp_mmi
+  @bitstreams//:bitstream_test_rom
+  @bitstreams//:bitstream_rom
+  @bitstreams//:rom_mmi
```

Comment:
Hmm, this presents a kind of bootstrapping problem. The `@bitstreams` workspace is not initialized on my  machine, so `ci/scripts/target-location.sh  @bitstreams//:bitstream_rom` gives me an empty string.

Reply:
Yeah, we still would need to get rid of the bad cache entry (both in GCP and locally). When we change these scripts, it should force a new bitstream cache entry as well, so rebasing will let everyone continue.

Reply:
Now that the cache is temporarily ok shouldn't a bazel sync --configure unblock local work as well?

Reply:
Ah, yes, if you created an entry for a newer hash and/or blew up the old one. :)